src/pydargs/__init__.py

```diff
@@ -135,7 +135,17 @@ def _create_parser(tp: Type[Dataclass]) -> ArgumentParser:
                 type=lambda x: field.type[x],
                 **kwargs,
             )
-
+        elif field.type is bytes:
+            encoding = field.metadata.get("encoding", "utf-8")
+            bytes_parser = partial(field.type, encoding=encoding)
+            setattr(bytes_parser, "__name__", encoding)
```

Comment:
is this just for the error message again?

Reply:
Indeed, it places the 'ascii' in

```
assert "argument --b: invalid ascii value: 'ꀀ'\n" in captured.err
```